src/SimpleChocolateyServer/Web.config

```diff
@@ -19,9 +19,14 @@
       <add name="ErrorMail" type="Elmah.ErrorMailModule, Elmah"/>
       <add name="ErrorFilter" type="Elmah.ErrorFilterModule, Elmah"/>
     </httpModules>
-    <httpRuntime maxRequestLength="31457280"/>
+    <httpRuntime maxRequestLength="2147482548"/>

```

Comment:
@ferventcoder if we are in doubt after changing this, we could do a quick test with a larger file.